python/pandemic_simulator/environment/pandemic_sim.py

```diff
@@ -322,3 +334,43 @@ def reset(self) -> None:
             regulation_stage=0,
             infection_above_threshold=False,
         )
+
+    def person_update(self, person: Person) -> None:
+        person.step(self._state.sim_time, self._contact_tracer)
+        return
+
+
+def prod_reduced(a: np.array, b: np.array, idx: list) -> tuple:
+    """
+    return AxB only at desired indices
+    """
+    return a[idx // b.size], b[idx % b.size]
+
+
+def comb2_reduced(l: np.array, idx: list) -> tuple:
```

Comment:
Could you improve the documentation on the arguments and what is happening inside the function?